### ISSUE TITLE
Quick replies UX issue with many options

### ISSUE THREAD

**Issue by ericnewcomer** [nyaruka/courier#373]
### Telegram
If a message has long or many quick replies, the Telegram implementation puts them all on one line and starts truncating. We should be smarter about breaking this into a keyboard with columnar options.

<img width="218" alt="Screen Shot 2021-08-12 at 11 21 54 AM" src="https://user-images.githubusercontent.com/211652/129248157-36926c4a-c3bf-481b-b1ab-1f90afc9557f.png">

### Viber
The inverse is true for Viber. In this case, the options are always present as a column with no mind to number of options or the length of the text.

<img width="218" alt="Screen Shot 2021-08-12 at 11 21 54 AM" src="https://user-images.githubusercontent.com/211652/129248629-0f2dda14-942b-4a05-93e2-853a8cc94c0d.png">

### Line
No example here, but a similar experience has also been reported here.

**Comment by matmsa27** [comment]
I would like to mention that [issue](https://github.com/rapidpro/rapidpro/issues/1491) here, its the same problem.

**Comment by rowanseymour** [comment]
@matmsa27 yeah I think we can use the same logic from here to optimize layout of Viber buttons
